src/client/pages/AssessmentSection/AssessmentSection.tsx

```diff
@@ -50,7 +53,9 @@ const AssessmentSection: React.FC<Props> = (props: Props) => {
 
       <SectionHeader assessmentName={assessmentName} sectionName={sectionName} disabled={disabled} />
 
-      {!panEuropean && <Descriptions sectionName={sectionName} descriptions={descriptions} disabled={disabled} />}
+      {!panEuropean && (
```

Comment:
also, let's remove this panEuropean check. we will allow editing panEuropean data